diff -ur ns-allinone-3.24.1/ns-3.24.1/wscript osx-ns-allinone-3.24.1/ns-3.24.1/wscript
--- ns-allinone-3.24.1/ns-3.24.1/wscript	2015-09-24 17:56:45.000000000 -0700
+++ osx-ns-allinone-3.24.1/ns-3.24.1/wscript	2016-02-23 12:02:39.000000000 -0800
@@ -332,7 +332,7 @@
         env.append_value('CCFLAGS', '-ftest-coverage')
         env.append_value('CXXFLAGS', '-fprofile-arcs')
         env.append_value('CXXFLAGS', '-ftest-coverage')
-        env.append_value('LINKFLAGS', '-lgcov')
+        #env.append_value('LINKFLAGS', '-lgcov')
         env.append_value('LINKFLAGS', '-coverage')
 
     if Options.options.build_profile == 'debug':
